SECURITY.md

```diff
@@ -2,7 +2,7 @@
 
 ## Supported Versions
 
-The current `master` and `release-pismo` branches are supported with security updates.
+The current `master` and only the latest `agoric-upgrade-*` tagged release and pre-release are supported with security updates.
```

Comment:
Note: I don't expect us to ever have a latest pre-release tag that is older (aka ancestor commit) than the latest release tag, so this works, but it initially worried me about opening ourselves to reports on outdated pre-release tags. More precisely, given our release process, the latest pre-release tag should always be the same revision as the latest release tag, or newer when we're about to release an upgrade.